habitat/tasks/rearrange/rearrange_sim.py

```diff
@@ -181,7 +195,7 @@ def reconfigure(self, config: Config):
         self._clear_objects(should_add_objects)
 
         self.prev_scene_id = ep_info["scene_id"]
-        self._viz_templates: Dict[float, int] = {}
+        self._viz_templates = {}
```

Comment:
Removing typing on purpose?

Reply:
The type is given line 72. Adding type again is redundant no?